deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -107,6 +107,10 @@ cephClusterSpec:
   # enable the ceph dashboard for viewing cluster status
   dashboard:
     enabled: true
+    annotations:
```

Comment:
Setting the annotations here is not an option. I believe instead what you want is to create the ingress [here in the helm chart](https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph-cluster/values.yaml#L319-L324), and see also [here](https://rook.github.io/docs/rook/v1.8/ceph-dashboard.html#ingress-controller) for more details.

Reply:
Could you please explain why it is not an option if it is included in the [template](https://github.com/rook/rook/blob/4f9a72c1db12862d89403276daa59df86cec6a0b/deploy/charts/rook-ceph-cluster/templates/ingress.yaml#L8)?

Reply:
My original link was invalidated by a recent merge... see [this values.yaml](https://github.com/rook/rook/blob/release-1.8/deploy/charts/rook-ceph-cluster/values.yaml#L318-L323) for where the `ingress.dashboard` settings are shown. The location in your PR is for `cephClusterSpec.dashboard.annotations`, which is different.

Reply:
Oh yes, my bad. It's under the `ingress.dashboard` now.